packages/react-components/react-tags/src/utils/useTagContextValues.ts

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react';
 import { TagContextValues, TagState } from '../components/Tag/index';
+import { TagButtonContextValues, TagButtonState } from '../TagButton';
 
-export function useTagContextValues_unstable(state: TagState): TagContextValues {
+export function useTagContextValues_unstable(
+  state: TagState | TagButtonState,
```

Comment:
```suggestion
  state: Pick<TagState, 'avatarSize' | 'avatarShape'>
```
nit: that should still work without needed to union both state types